packages/perspective-workspace/src/js/workspace/workspace.js

```diff
@@ -411,15 +411,16 @@ export class PerspectiveWorkspace extends DiscreteSplitPanel {
             await widget.toggleConfig();
         }
 
+        this.detailPanel.close();
```

Comment:
I think this will introduce a bug, where the detail panel is closed when there's already a global filter widget

Reply:
Sorry, this was a commit from another PR.  Good measure we have code review!